lib/modules/versioning/debian/index.ts

```diff
@@ -0,0 +1,171 @@
+import type { RangeStrategy } from '../../../types';
+import { DistroInfo, DistroInfoRecordWithVersion } from '../distro';
+import { GenericVersioningApi } from '../generic';
+import type { GenericVersion } from '../generic';
+import type { NewValueConfig, VersioningApi } from '../types';
+
+export const id = 'debian';
+export const displayName = 'Debian';
+export const urls = [
+  'https://debian.pages.debian.net/distro-info-data/debian.csv',
+];
+export const supportsRanges = true;
+export const supportedRangeStrategies: RangeStrategy[] = ['pin'];
+
+const RELEASE_PROP = 'release';
+
+class RollingReleasesData {
```

Comment:
Can we move this to a `./common.ts` file, so reducing this file size for better maintainabillity?